manifests/backend.pp

```diff
@@ -51,6 +51,44 @@
 # @param license_content
 #   The content of sensu-go enterprise license
 #   Do not define with license_source
+# @param ad_auths
+#   Hash of sensu_ad_auth resources
+# @param assets
+#   Hash of sensu_asset resources
+# @param checks
+#   Hash of sensu_check resources
+# @param cluster_members
+#   Hash of sensu_cluster_member resources
+# @param cluster_role_bindings
+#   Hash of sensu_cluster_role_binding resources
+# @param cluster_roles
+#   Hash of sensu_cluster_role resources
+# @param configs
+#   Hash of sensu_config resources
+# @param entities
+#   Hash of sensu_entitie resources
+# @param events
+#   Hash of sensu_event resources
+# @param filters
+#   Hash of sensu_filter resources
+# @param handlers
+#   Hash of sensu_handler resources
+# @param hooks
+#   Hash of sensu_hook resources
+# @param ldap_auths
+#   Hash of sensu_ldap_auth resources
+# @param mutators
+#   Hash of sensu_mutator resources
+# @param namespaces
+#   Hash of sensu_namespace resources
+# @param role_bindings
+#   Hash of sensu_role_binding resources
+# @param roles
+#   Hash of sensu_role resources
+# @param silencing
```

Comment:
please use `silencings` with a `s` to ensure consistency

Reply:
Fixed.